### ISSUE TITLE
dxtbx Internal Error: dxtbx/model/scan.h(350): DXTBX_ASSERT(image_range_[1] + 1 == rhs.image_range_[0]) failure.

### ISSUE THREAD

**Issue by graeme-winter** [cctbx/dxtbx#367]
This should not be emitted

```
Silver-Surfer tmp :) $ dials.import /Users/graeme/git/dials/build/dials_data/x4wide/X4_wide_M1S4_2_000[0-9].cbf /Users/graeme/git/dials/build/dials_data/x4wide/X4_wide_M1S4_2_002[0-9].cbf
DIALS (2018) Acta Cryst. D74, 85-97. https://doi.org/10.1107/S2059798317017235
DIALS 3.dev.458-g0aadf770f
dxtbx Internal Error: /Users/graeme/git/dials/modules/dxtbx/model/scan.h(350): DXTBX_ASSERT(image_range_[1] + 1 == rhs.image_range_[0]) failure.
The following parameters have been modified:

input {
  experiments = <image files>
}

--------------------------------------------------------------------------------
  format: <class 'dxtbx.format.FormatCBFMiniPilatus.FormatCBFMiniPilatus'>
  num images: 19
  sequences:
    still:    0
    sweep:    2
  num stills: 0
--------------------------------------------------------------------------------
Writing experiments to imported.expt
```

if I import those images, I want those images importing

**Comment by graeme-winter** [comment]
We are using an assert in here as _part of the control flow_ which I think cannot be a good thing

**Comment by graeme-winter** [comment]
16 months later oh right found the print -> will be fixed by #567 so closing in the meanwhile
